torch_np/_funcs_impl.py

```diff
@@ -730,50 +709,29 @@ def triu_indices(n, k=0, m=None):
 def tril_indices_from(arr: ArrayLike, k=0):
     if arr.ndim != 2:
         raise ValueError("input array must be 2-d")
-    result = torch.tril_indices(arr.shape[0], arr.shape[1], offset=k)
-    return tuple(result)
+    # Return a tensor rather than a tuple to avoid a graphbreak
+    return torch.tril_indices(arr.shape[0], arr.shape[1], offset=k)
```

Comment:
This is a break w.r.t. numpy, so should be added to the list in https://github.com/Quansight-Labs/numpy_pytorch_interop/issues/73 I guess.
Also this should fail on CI, so those tests need to be adjusted. 

Semantics wise, should we not be worried about

```
In [40]: a = np.arange(6).reshape(3, 2)

In [41]: idx = np.tril_indices_from(a)

In [42]: a[idx]
Out[42]: array([0, 2, 3, 4, 5])

In [43]: a[np.asarray(idx)]
Out[43]: 
array([[[0, 1],
        [2, 3],
        [2, 3],
        [4, 5],
        [4, 5]],

       [[0, 1],
        [0, 1],
        [2, 3],
        [0, 1],
        [2, 3]]])
```

Reply:
Added to that issue.

Reply:
>ISTM the main usage of these is to actually index something with the result, so if we return a tensor, the next stage for a user is to swear and throw an explicit tuple() call. So we're back to the same graph break and only earn a swear from a user

As discussed in this example, we should be fine, as it's possible to index also with a tensor while preserving the semantics.